src/scripts/OSFramework/Helper/Device.ts

```diff
@@ -335,10 +342,10 @@ namespace OSFramework.Helper {
 		public static get IsIphoneWithNotch(): boolean {
 			if (DeviceInfo._isIphoneWithNotch === undefined) {
 				// get the device pixel ratio
-				const ratio = window.devicePixelRatio || 1;
+				const ratio = (sessionStorage.pixelRatio ? sessionStorage.pixelRatio : window.devicePixelRatio) || 1;
 				const currScreen: iphoneDetails = {
-					width: window.screen.width * ratio,
-					height: window.screen.height * ratio,
+					width: window.visualViewport.width * ratio,
+					height: window.visualViewport.height * ratio,
```

Comment:
Why was this change?

Reply:
This was one of the errors in the console, aligned with Bernardo Cardoso about this fix. Because the screen was null at the calculation time.